docs/ref/models/querysets.txt

```diff
@@ -2089,6 +2089,39 @@ instance (if the database normally supports it).
 
     The ``ignore_conflicts`` parameter was added.
 
+``bulk_save()``
+~~~~~~~~~~~~~~~
+
+.. versionadded: 2.2
+
+.. method:: bulk_save(objs, fields, batch_size=None)
+
+This method updates the given fields on the provided list of objects in the
+database in an efficient manner (generally only one query)::
+
+    >>> objects = [
+    ...    Entry.objects.create(headline='Entry 1'),
+    ...    Entry.objects.create(headline='Entry 2')
```

Comment:
To *really* nitpick, there could be a comma here too... :stuck_out_tongue:

Reply:
😂 fixed